reana_commons/utils.py

```diff
@@ -198,8 +198,13 @@ def get_workspace_disk_usage(workspace, summarize=False, block_size=None):
         command.append("-s")
     else:
         command.append("-a")
-    command.append(workspace)
-    disk_usage_info = subprocess.check_output(command).decode().split()
+    command.append(directory)
+    return subprocess.check_output(command).decode().split()
+
+
+def get_workspace_disk_usage(workspace, summarize=False, block_size=None):
```

Comment:
What do you think of moving these functions to the [User](https://github.com/reanahub/reana-db/blob/c89d3703b8bcae2f8af83f6c81473b201e1997a0/reana_db/models.py#L45) and [Workflow](https://github.com/reanahub/reana-db/blob/c89d3703b8bcae2f8af83f6c81473b201e1997a0/reana_db/models.py#L238) models? I think that the usage (e.g. [R-Server PR](https://github.com/reanahub/reana-server/pull/281/files)) would be more intuitive.

Reply:
Good idea, but that would mean we need to inject `shared volume path` somewhere in `reana-db` to get the absolute path. Other option would be to use relative paths in db models and append it with shared volume path in `reana-commons (get_disk_usage)`. Which would be a better way to go?

Reply:
I would say the second option. We can work with relative paths in REANA-DB and then REANA-Commons figures out the path prefix because it is the component that knows about shared volumes.

Something that can help to shape how this looks is thinking that at some point:
- We might switch workspaces from Ceph to EOS
    - In this case the change isn't big, it is just that the base path would be an EOS path
- We might want to use S3 interfaces instead of a FUSE filesystem
    - In this case the concept of base path will be different. It will be first of all a server (configuration), and then a bucket where the data will be stored, e.g. https://s3fs.readthedocs.io/en/latest/#examples

In both cases, the low level storage details will live only in R-Commons.